tests/test_dle.py

```diff
@@ -404,49 +404,6 @@ def test_cache_blocking_hierarchical(blockshape0, blockshape1, exception):
 
 @pytest.mark.parametrize("blockinner", [False, True])
 def test_cache_blocking_imperfect_nest(blockinner):
-    """
```

Comment:
redundant?

Reply:
Never mind, and thanks for making me notice: I had originally removed it because, yes, as I said, I was going to move it to test_mpi; but then I rolled most of the temporary edits back; hence it does make sense to keep this test where it was; so, reinstated.